client/api/omni/specs/omni.proto

```diff
@@ -1103,3 +1103,17 @@ message KubernetesNodeAuditResultSpec {
   // DeletedNodes contains the list of nodes that were last removed.
   repeated string deleted_nodes = 1;
 }
+
+// MachineRequestSetSpec creates the number of machine requests.
+// Can be either connected to a machine class or be manual.
+message MachineRequestSetSpec {
+  string provider_id = 1;
+  int32 machine_count = 2;
+
+  string talos_version = 3;
+  string schematic_id = 4;
```

Comment:
I'm not sure about using raw schematic here though.
Maybe we should use the list of extensions, kernel args and meta args here and in the `MachineRequest` resource.